dispatchers.go

```diff
@@ -201,14 +222,29 @@ func dispatchCopy(b *Builder, args []string, attributes map[string]bool, flagArg
 			return fmt.Errorf("COPY only supports the --chmod=<permissions> --chown=<uid:gid> and the --from=<image|stage> flags")
 		}
 	}
-	b.PendingCopies = append(b.PendingCopies, Copy{From: from, Src: args[0:last], Dest: dest, Download: false, Chown: chown, Chmod: chmod})
+	var files []File
+	for _, heredoc := range heredocs {
+		command := parseCommandForHeredoc(original)
+		content := heredoc.Content
+		if heredoc.Chomp {
+			content = buildkitparser.ChompHeredocContent(content)
+		}
+		file := File{
+			Data:    content,
+			Name:    heredoc.Name,
+			Command: command,
```

Comment:
Afaics `heredoc.FileDescriptor` is defined in buildkit struct but even buildkit never uses it ( correct me if i am looking code wrongly ) but I have added it to imagebuilder and buildah can implement it even if buildkit does not implements it.

I am using `heredoc.Expand` in imagebuilder itself for expansion of heredoc instead of letting buildah do it, in function `processHereDocs` in `dispatcher.go`